nntrainer/tensor/manager.cpp

```diff
@@ -625,13 +625,51 @@ Manager::getWeights(const std::function<bool(const Weight *)> &condition) {
 }
 
 void Manager::flushCache() {
-  weight_pool.flushCache();
-  tensor_pool.flushCache();
+  if (!swap_lookahead) {
+    weight_pool.flushCache();
+    tensor_pool.flushCache();
+  }
 }
 
 void Manager::flushCacheExcept(unsigned int order) {
-  weight_pool.flushCacheExcept(order);
-  tensor_pool.flushCacheExcept(order);
+  auto loadAsync = [&](TensorPool &pool, unsigned int order) {
```

Comment:
Not in this PR, but this ```flushCacheExecpt```is called before every execution order. forwarding, calcGradient, calcDerivatives. But for the backwarding, this is called in neuralnet.cpp, and for the forwarding, it is called in network_graph.cpp. So.. how about making forwrading_op like backwarding_op in nerualnet.cpp, and keeping the consistence?

Reply:
Good point. I'll modify for forwarding_op in next PR.